.github/CODEOWNERS

```diff
@@ -79,13 +79,13 @@
 /sdk/communication/azure-communication-sms/                          @RoyHerrod @arifibrahim4
 
 # PRLabel: %Communication - Identity
-/sdk/communication/azure-communication-identity/                     @Azure/acs-identity-sdk @petrsvihlik @AikoBB @maximrytych-ms @ostoliarova-msft
+/sdk/communication/azure-communication-identity/                     @Azure/acs-identity-sdk @petrsvihlik @AikoBB @maximrytych-ms @ostoliarova-msft @mjafferi-msft
 
 # PRLabel: %Communication - NetworkTraversal
 /sdk/communication/azure-communication-networktraversal/             @ajpeacock0 @nathpete-msft
 
 # PRLabel: %Communication - Common
-/sdk/communication/**/_shared/                                       @Azure/acs-identity-sdk @petrsvihlik @AikoBB @maximrytych-ms @ostoliarova-msft
+/sdk/communication/**/_shared/                                       @Azure/acs-identity-sdk @petrsvihlik @AikoBB @maximrytych-ms @ostoliarova-msft @mjafferi-msft
```

Comment:
I think we provided here the wrong path, should it be as follow?
`/sdk/communication/**/**/**/**/_shared/`

Reply:
also, do we have to add here the shared folder under the `test` folder as well?

Reply:
Yes, the path for the shared folder should be : /sdk/communication/**/**/**/**/_shared/

Reply:
let's wait for other's response for the shared folder within the test folder 
cc: @petrsvihlik @maximrytych-ms @ostoliarova-msft

Reply:
I was under the impression that `**/_shared/` matches the `_shared` folder at any depth....

https://stackoverflow.com/a/67224003/1332034
![image](https://user-images.githubusercontent.com/9810625/202654285-a7f66700-3973-4cfb-9a46-4a9b72b21da6.png)

@jsquire do you know if that's the case?

Reply:
@mjafferi-msft could you please create a testing (draft) PR with a change just in the _shared folder to see if the right people are included? once verified, we'll close that PR without merging.

Reply:
works correctly as it is now - #27665